cmd/viewportdemo/main.go

```diff
@@ -0,0 +1,112 @@
+package main
```

Comment:
The viewport widget and demo **really need to go**. Aside from being unfinished, it relies on Fyne's renderer to do clipping which means it leaks VRAM very badly. The more you zoom in, the more memory is needed for generating the textures from the scene.

Reply:
Removed